docs/src/tutorials/software-management.md

```diff
@@ -70,51 +41,30 @@ More info about managing the device software can be found [here](https://cumuloc
    - to add a new software, enter a name for the software (and confirm it by clicking Create new in the resulting window),
      a description and its version.
    - to add a new version, select the software for which you want to add a new version from the dropdown list in the Software
-     field and enter a version and plugin type as **version::apt** (In the picture below the version is empty and are optional)
+     field and enter a version of the package. The version is optional and can be left with a white space, meaning the latest version.
+   > Note: The version field format is `package_version::plugin_type_name`.The plugin type name that is provided here is used to pick
```

Comment:
Wondering if this note is needed here until we support multiple plugins.

Reply:
Now this will not confuse the users if we even add a new plugin. Its to give more details to the user.